source/extensions/common/tap/tap_config_base.cc

```diff
@@ -105,21 +106,24 @@ TapConfigBaseImpl::TapConfigBaseImpl(const envoy::config::tap::v3::TapConfig& pr
     // extract message validation visitor from the context and use it to define config
     ProtobufTypes::MessagePtr config;
     if (absl::holds_alternative<TsfContextRef>(context)) {
+      ENVOY_LOG_MISC(debug, "{}: Transport socket context", __func__);
```

Comment:
These debug statements seem to be somewhat noisy (specifically because `createSinkPtr` also has a debug statement). Consider removing these, and also consider changing some to trace level, if they do not add much debug information.

Reply:
Nice comment and removed.